crates/epaint/src/tessellator.rs

```diff
@@ -885,8 +898,13 @@ fn stroke_path(
     let idx = out.vertices.len() as u32;
 
     // expand the bounding box to include the thickness of the path
-    let bbox = Rect::from_points(&path.iter().map(|p| p.pos).collect::<Vec<Pos2>>())
-        .expand((stroke.width / 2.0) + feathering);
+    let bbox = Rect::from_points(
+        &path
+            .iter()
+            .map(|p| translate_stroke_point(p, stroke).pos)
```

Comment:
I think running `translate_stroke_point` once on all the points (mutating the input) would be simpler, and also fix the gap between the stroke and the fill

Reply:
That did simplify things but it didn't completely close the gap. The issue is that feathering is slightly shrinking both the fill and stroke in opposite directions. The gap does disappear when feathering is disabled. I haven't looked into it but I'm wondering if you have any suggestions.

1px feathering
![Screenshot 2024-08-28 at 21 49 07](https://github.com/user-attachments/assets/5b3a090d-7e50-4aaf-b5d7-cb622c91ccb6)

5px feathering
![Screenshot 2024-08-28 at 21 55 09](https://github.com/user-attachments/assets/f1e5328a-4029-4630-aed2-d9fb3de2cbe5)

No feathering
![Screenshot 2024-08-28 at 21 54 36](https://github.com/user-attachments/assets/0ca3ccc4-7676-4b87-ad1b-6b7d2034bcc8)

Reply:
I spent some time thinking about this and landed on: the fill should feather to the stroke color (instead of transparent), and vice versa.

I first thought that it would be okay to just grow the fill but that wouldn't work if the stroke is translucent.